packages/tpetra/core/src/Tpetra_Distributor.cpp

```diff
@@ -46,49 +46,14 @@
 #include <numeric>
 
 namespace Tpetra {
-  namespace Details {
-    std::string
-    DistributorSendTypeEnumToString (EDistributorSendType sendType)
-    {
-      if (sendType == DISTRIBUTOR_ISEND) {
-        return "Isend";
-      }
-      else if (sendType == DISTRIBUTOR_RSEND) {
-        return "Rsend";
-      }
-      else if (sendType == DISTRIBUTOR_SEND) {
-        return "Send";
-      }
-      else if (sendType == DISTRIBUTOR_SSEND) {
-        return "Ssend";
-      }
-      else {
-        TEUCHOS_TEST_FOR_EXCEPTION(true, std::invalid_argument, "Invalid "
-          "EDistributorSendType enum value " << sendType << ".");
-      }
-    }
-
-    std::string
-    DistributorHowInitializedEnumToString (EDistributorHowInitialized how)
-    {
-      switch (how) {
-      case Details::DISTRIBUTOR_NOT_INITIALIZED:
-        return "Not initialized yet";
-      case Details::DISTRIBUTOR_INITIALIZED_BY_CREATE_FROM_SENDS:
-        return "By createFromSends";
-      case Details::DISTRIBUTOR_INITIALIZED_BY_CREATE_FROM_RECVS:
-        return "By createFromRecvs";
-      case Details::DISTRIBUTOR_INITIALIZED_BY_CREATE_FROM_SENDS_N_RECVS:
-        return "By createFromSendsAndRecvs";
-      case Details::DISTRIBUTOR_INITIALIZED_BY_REVERSE:
-        return "By createReverseDistributor";
-      case Details::DISTRIBUTOR_INITIALIZED_BY_COPY:
-        return "By copy constructor";
-      default:
-        return "INVALID";
-      }
-    }
-  } // namespace Details
+  // We set default values of Distributor's Boolean parameters here,
+  // in this one place.  That way, if we want to change the default
+  // value of a parameter, we don't have to search the whole file to
+  // ensure a consistent setting.
+  namespace {
+    // Default value of the "Debug" parameter.
+    const bool tpetraDistributorDebugDefault = false;
```

Comment:
Do we not need default value for barrierBetween_default or useDistinctTags_default any more?  They were set in old code.

Ah, I see below -- they moved to Details; never mind.